Source/WebCore/platform/graphics/texmap/TextureMapper.cpp

```diff
@@ -505,6 +505,37 @@ void TextureMapper::drawTexture(const BitmapTexture& texture, const FloatRect& t
     drawTexture(texture.id(), texture.colorConvertFlags() | (texture.isOpaque() ? OptionSet<TextureMapperFlags> { } : TextureMapperFlags::ShouldBlend), targetRect, matrix, opacity, allEdgesExposed);
 }
 
+void TextureMapper::drawTextureFragment(const BitmapTexture& sourceTexture, const FloatRect& sourceRect, const FloatRect& targetRect)
```

Comment:
the name could be better as "fragment" may be misleading (towards fragment shader)

Reply:
actually it's hard to find a better name...